src/main/java/de/tum/in/www1/exerciseapp/web/rest/ExerciseResource.java

```diff
@@ -246,7 +246,7 @@ public ExerciseResource(ExerciseRepository exerciseRepository, ExerciseService e
      * @return the ResponseEntity with status 200 (OK)
      */
     @DeleteMapping(value = "/exercises/{id}/participations")
-    @PreAuthorize("hasAnyRole('ADMIN', 'TA')")
+    @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
```

Comment:
TA is missing.

Reply:
Reset feature is currently not in the list of features. I've added it now to the tables with the access right 'TA', hopefully that's correct.